File: build-tests/ObjcTest/ObjcTest/ViewController.m
//
//  ViewController.m
//  ObjcTest
//
//  Created by Ildar Abdullin on 7/22/20.
//  Copyright © 2020 Okta. All rights reserved.
//

#import "ViewController.h"
@import OktaOidc;

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    OktaOidc *oidc = [[OktaOidc alloc] initWithConfiguration:nil error:nil];
    NSLog(@"%@", @(oidc.hasActiveBrowserSession));
}


@end

File: biometric-storage/Biometric storage/ViewController.swift
//
//  ViewController.swift
//  Biometric storage
//
//  Created by Ildar Abdullin on 3/15/19.
//  Copyright © 2019 Okta. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var storeStackView: UIStackView!
    @IBOutlet weak var biometricImageView: UIImageView!
    @IBOutlet weak var readStackView: UIStackView!
    @IBOutlet weak var storeButton: UIButton!
    @IBOutlet weak var passwordTextField: UITextField!
    
    override func viewDidLoad() {
        super.viewDidLoad()

        readStackView.isHidden = true
    }


}

